Formula/unnethack.rb

```diff
@@ -1,8 +1,9 @@
 class Unnethack < Formula
   desc "Fork of Nethack"
   homepage "https://unnethack.wordpress.com/"
-  url "https://github.com/UnNetHack/UnNetHack/archive/5.3.2.tar.gz"
-  sha256 "a32a2c0e758eb91842033d53d43f718f3bc719a346e993d9b23bac06f0ac9004"
+  url "https://github.com/UnNetHack/UnNetHack/archive/6.0.3-09d501423.tar.gz"
+  version "6.0.3-09d501423"
```

Comment:
This version scheme makes this sound like a pre-release. In fact, they've since added a new tag: `6.0.3-bf0508699`.

Reply:
We should probably add a `livecheck` block with the default regex for tags.